src/Console/Command/ConfigGeneratorCommand.php

```diff
@@ -15,7 +15,7 @@
 
 final class ConfigGeneratorCommand extends Command
 {
-    public const CONFIG_FILE_NAME = 'crunz.yml';
+    public static string $CONFIG_FILE_NAME = 'crunz.yml';
```

Comment:
It will be better to leave that constant as is and create new parameter in container.